tests/end_to_end/test_vulnerability_detector/cases/test_vulnerability.yaml

```diff
@@ -250,33 +250,33 @@
         amd64: luxon-2.5.2
         arm64v8: luxon-2.5.2
       centos:
-        amd64: grafana-8.5.27-1
-        arm64v8: grafana-8.5.27-1
+        amd64: grafana-9.3.16
+        arm64v8: grafana-9.3.16
       ubuntu:
-        amd64: grafana-8.5.27
-        arm64v8: grafana-8.5.27
+        amd64: grafana-9.3.16
+        arm64v8: grafana-9.3.16
   body:
     operation: update_package
     package:
       from:
         centos:
-          amd64: grafana-8.5.27-1
-          arm64v8: grafana-8.5.27-1
+          amd64: grafana-9.3.16
+          arm64v8: grafana-9.3.16
         ubuntu:
-          amd64: grafana-8.5.27
-          arm64v8: grafana-8.5.27
+          amd64: grafana-9.3.16
+          arm64v8: grafana-9.3.16
         windows:
           amd64: node-v18.20.2
         macos:
           amd64: luxon-2.5.2
           arm64v8: luxon-2.5.2
       to:
         centos:
-          amd64: grafana-9.1.1-1
-          arm64v8: grafana-9.1.1-1
+          amd64: grafana-9.4.1
+          arm64v8: grafana-9.4.1
```

Comment:
```suggestion
          amd64: grafana-9.4.1-1
          arm64v8: grafana-9.4.1-1
```

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5719/commits/6ec247eddca368da3903957c75b01715836fce30